examples/tests/Containers.ttl

```diff
@@ -1,6 +1,6 @@
 @prefix elmo: <http://bp4mc2.org/elmo/def#>.
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
-@prefix container: <http://localhost:8080/container/>.

```

Comment:
Should not be deleted, but an inserted @prefix ldtcontainer. And duplicate the containerdefinitions (not very nice, but workable)